types/hasbin/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for hasbin 1.2
+// Project: https://github.com/springernature/hasbin
+// Definitions by: Micha Streppel <https://github.com/michastreppel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function hasbin(bin: string, resultFunction: (result: boolean) => void): void;
```

Comment:
This module doesn't create a global. Please follow the standard format for CommonJS packages.

Reply:
Could you please tell me where I can find this standard format? I cannot seem to find it. Thanks in advance